### PR TITLE
DDP/Set up a monitoring system: Update week 3 progress

### PR DESCRIPTION
**Pull request by jayanth-kumar-morem**
## Project

Name of the project

### Changes

What was updated (new pages, weekly update, etc.)

### Notes

If there are any further notes.

### CODE REVIEW

File: docs/2023/DDP/Set up a monitoring system/updates/2023-07-21.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ author: Jayanth Kumar
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
+- [ ] Install and configure Grafana for infrastructure metrics monitoring on EC2.
```
Comment (photon0205) [inline docs/2023/DDP/Set up a monitoring system/updates/2023-07-21.md:7]: add `x` inside the `[ ]` for the milestones you have achieved

File: docs/2023/DDP/Set up a monitoring system/updates/2023-07-21.md [hunk 2]
```diff
@@ -4,13 +4,13 @@ author: Jayanth Kumar
...
+- [ ] Set up data collection for CPU, RAM, and Disk usage metrics.
+- [ ] Design and create interactive dashboards for visualizing the metrics.
+- [ ] Document the entire configuration and dashboard setup process.
 
-## Screenshots / Videos 
-
-## Contributions
```
Comment (photon0205) [inline docs/2023/DDP/Set up a monitoring system/updates/2023-07-21.md:14]: Also what about contributions? Why haven't you added the contributions?

**Review by photon0205 — CHANGES_REQUESTED** [review]

File: docs/2023/DDP/Set up a monitoring system/updates/2023-07-21.md (hunk 2 above)
Comment (jayanth-kumar-morem) [reply docs/2023/DDP/Set up a monitoring system/updates/2023-07-21.md:14]: My work is completely about setting up services on the aws ec2 instances.

File: docs/2023/DDP/Set up a monitoring system/updates/2023-07-21.md (hunk 1 above)
Comment (jayanth-kumar-morem) [reply docs/2023/DDP/Set up a monitoring system/updates/2023-07-21.md]: Ack

**Pull request merged**